lighthouse-cli/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": false,
```

Comment:
We had an intern fix up Angular's code base for `any` using a semi-automated process based on tslint. Reach out to Igor Minar or Alex Eagle for details. Though that's probably for your relatively few issues.